openapi-3-2/index.d.ts

```diff
@@ -283,7 +283,7 @@ type Response = {
   links?: Record<string, Link | Reference>;
 };
 
-type Callbacks = Record<string, PathItem | Reference>;
+type Callbacks = Record<string, PathItem>;
```

Comment:
`Callbacks` is supposed to be the type for the "callbacks" field Components and Operations. I think what's actually needed is this,

```suggestion
type Callback = Record<string, PathItem>;
type Callbacks = Record<string, Callback | Reference>;
```